numba/tests/test_typedlist.py

```diff
@@ -530,6 +533,60 @@ def test_repr(self):
         expected = "ListType[int32]([1, 2, 3])"
         self.assertEqual(expected, repr(l))
 
+    def test_repr_long_list_ipython(self):
+        # Test repr of long typed Lists in an IPython session
+        base_cmd = [sys.executable, '-m', 'IPython']
+        base_cmd += ['--quiet', '--quick', '--no-banner', '--colors=NoColor']
+        try:
+            ver = subprocess.check_output(base_cmd + ['--version'])
+        except subprocess.CalledProcessError as e:
+            self.skipTest("ipython not available: return code %d"
+                          % e.returncode)
```

Comment:
I see this is very similar to the code in : https://github.com/numba/numba/blob/master/numba/tests/test_dispatcher.py#L1418-L1425 I wonder if this can be turned into something re-usable. Perhaps to be added to: https://github.com/numba/numba/blob/master/numba/tests/support.py

Reply:
To clarify: the question is if this is a good idea or even possible? If it turns out, that attempting a re-use unnecessarily complicates matters that insight will have value in itself and the code can stay the way it is. Hope that is clear?

Reply:
Thank you for your review. We could wrap this up as a separate method, like `check_ipython_available`, but I wonder if it's worth the trouble. After all, we're just using it in only two places.

Reply:
@esc Can I mark this one as resolved?

Reply:
OK, fair enough, maybe next time someone touches this.